week3/Borko/memory_profiler.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import os
+import cProfile
+from pyprof2calltree import convert
+from functools import wraps
+import time
+
+
+def profile(func):
+    """
+    This is finction decorator for profiling other functions
+    """
+    @wraps(func)
+    def wrap_function(*args, **kwargs):
+        pr = cProfile.Profile()
+        pr.enable()
+        result = func(*args, **kwargs)
+        pr.disable()
+        convert(pr.getstats(), "ptof.tree")
+        print(pr.print_stats(sort="time"))
```

Comment:
pr.print_stats() prints, no need to print the print
In your output you have printed `None` which is because of this

Reply:
Ok